infra/src/main/resources/i18n/translations.fi.json

```diff
@@ -91,7 +91,8 @@
             "invalid-path": "Virhe: pyynnön polku oli virheellinen ({{method}})",
             "missing-parameter": "Virhe: pyynnöstä puuttuu vaadittu parametri ({{param}})",
             "invalid-body": "Virhe: pyynnön sisältöä ei voitu lukea",
-            "conversion-failed": "Virhe: Pyynnössä oli virheellinen arvo {{name}}"
+            "conversion-failed": "Virhe: Pyynnössä oli virheellinen arvo {{name}}",
+            "invalid-version": "Virhe: Käyttöliittymän versio ei vastaa palvelimen versiota"
```

Comment:
Ajattelin, että "palvelimen"-termi olisi tunnetumpi verrattuna esim. rajapinnan/palvelun/tms-termeihin. Muita ehdotuksia?

Reply:
Tuleekos tämä lukemaan tyyliin johonkin failaavan kutsun virhetoastiin tms.? Mietin olisiko joku hieman fuzzympi ja ohjeellinen versio parempi, tyyliin vaikka jotain tyyliin "Geoviitteen versio on päivittynyt taustalla, lataa sivu uudestaan"

Reply:
Teoriassa voi tulla toastiinkin, mutta nythän näytetään se "versio on päivittynyt"-dialogi eikä toastia. (Dialogi piirtyy toastien päälle tms) 

Oikeastaanhan toi virheviesti on tällä tavoin täsmällisempikin, sillä sehän bäkkäri vaan vertaa että mätsääkö request headerissä oleva versio vai ei (eikä niinkään että kumpi on uudempi/vanhempi)🤔

Reply:
Täsmällisempi joo, lähinnä mietin tuota siltä kannalta että mitä itse miettisin käyttäjänä jos näkisin tuollaisen virheilmoituksen (varmaan olettaisin liittyvän johonkin spesifiin operaatioon enkä koko Geoviitteen tilaan, vähän niinkuin melkein kaikki muutkin toastit.) Mutta jos näitä ei oikeastaan näytetä käyttäjille, niin silloin tämä on mielestäni ihan fine.